tests/test_formatting_utils.py

```diff
@@ -0,0 +1,49 @@
+import tokenize
+
+from pydocstringformatter._formatting._utils import _compare_formatters
+from pydocstringformatter._formatting.base import Formatter
+
+
+class MakeAFormatter(Formatter):
```

Comment:
We might want to create a `test/_test_utils.py` file so we don't have to duplicate this code.

Reply:
good idea